v22.2/ui-schedules-page.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Schedules Page
+summary: The Schedules page of the DB Console provides details about scheduled tasks performed by your cluster.
+toc: true
+docs_area: reference.db_console
+---
+
+{% include_cached new-in.html version="v22.2" %} The **Schedules** page of the DB Console provides details about the scheduled tasks performed by your cluster. These can include:
+
+- [Scheduled backups](manage-a-backup-schedule.html)
+- [Scheduled auto statistics compaction](cost-based-optimizer.html#table-statistics)
```

Comment:
and row level ttl schedules (see slide 22 - https://docs.google.com/presentation/d/1YiCJkd7Mr155skn1HPA_0IHlrns-jAojDEpkIhFqmY4/edit?usp=sharing)

Reply:
@livlobo Yes! good point! I made a note of this yesterday after watching your session — I will add this in. Also, I was going to merge the scheduled backup PTS docs with Aditya and then mention that in this PR since the PTS records show up on the schedule ID page when present. We were thinking of saying that this is an internal mechanism that required no configuring by the user, but here are the details of what it is.